2022/src/yaml/data.yaml

```
@@ -68,19 +69,90 @@ session_types:
     timetable_style: table
 sessions:
   - begin_at: 09:00
+    description: |
+      Active Directory の Kerberos 認証を利用した攻撃をツールを用いて攻撃を行います。
+      また、ツールを用いた Windows Event Log の解析についてもデモを行います。
     end_at: 12:00
+    lessons:
+      - Active Directory における攻撃
+      - Kerberos 認証に対してのツールを用いた攻撃
+    level: 中級
     name: Kerberos さんは、コミュ症です。
+    preparations:
+      - インターネットへの接続環境
+      - 16GB ほどのメモリがあるホスト PC (下記仮想環境構築のため)
+      - VirtualBox 環境
+      - VirtualBox 内に Detection Lab を用いた AD 環境の構築
     room: Hands-on 1
+    speakers:
+      - description: |
+          NTT グループの社員。
+        member_of: NTT アドバンステクノロジ
+        name: 古市 昌弘
+    tags:
+      - OSS
+      - Security
     type: Hands-on
   - begin_at: 09:30
+    description: |
+      本ハンズオンではコンテナオーケストレーションツールである k8s の運用に関わる問題について、E2E テストにフォーカスしクラスタ構成のテストを構築する方法を学習します。
+      <br />
+      プログラム (予定) : モニタリングとの違い、Terratest・Ginkgoなどについて、クラスタテストの実装、クラスタアップデートとテスト実施
     end_at: 11:30
+    lessons:
+      - k8s の運用に役立つクラスタテストのノウハウやツールについて
+    level: 初級
     name: k8s 中級者向けハンズオン
+    preparations:
+      - インターネット接続できる環境で PC は SSH 接続可能なこと
+      - ビデオ会議可能なスペックを保有していること
+    requirements:
+      - k8s の基本的なリソースや構成するコンポーネントについて理解していること
+      - Linux の基本的なコマンドを理解していること
+      - 業務や趣味でプログラミング経験があること (演習では Go 言語を利用しますが、Go 言語に対する理解は無くても問題ありません)
     room: Hands-on 2
+    speakers:
+      - description: |
+          イノベーションセンター GPU-k8s PJ 所属。
+        member_of: NTT コミュニケーションズ
+        name: 原田 和明
+    tags:
+      - Cloud
+      - Container
+      - OSS
+      - Programming
     type: Hands-on
   - begin_at: 10:00
+    description: |
+      デザイン思考の鍵となる「ペルソナ」。
+      このペルソナについて、実際にワークショップ内でペルソナをつくる体験を通して、
+      (1) ターゲットとペルソナの違い
+      (2) ペルソナをつくるメリット
+      (3) ペルソナをつくるときのコツは何か?
+      を学んでいきます。
+      <br />
+      Web 会議 (Teams または Zoom) でディスカッションしながら Miro を操作してペルソナをつくります。
+      【パソコンからのアクセス】または【スマホかタブレットで Web 会議 + パソコンで Miro を操作】という形で、Web 会議と Miro が同時に使用できる形で接続をお願いします。
+      <br />
+      「観察のみ」という方はノイズになるので参加をご遠慮ください。
```

「観察のみ」というのがよくわからなかったので書き直すとこんな感じでしょうか……？ノイズになるという表現はちょっときつそうな気がしたのでちょっと削除する方向でsuggestします

```suggestion
      「観察のみ」(音声によるディスカッションができない方)は参加をご遠慮ください。
```

---

修正しました

---

確認しました。修正ありがとうございます。問題ないと思います。